modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -1024,22 +931,17 @@ test(
 		});
 
 		await test.step('Check that My Page appears as exit pages and the number of views', async () => {
-			await expect(page.getByText('My Page - Lifer...')).toBeVisible({
-				timeout: 100 * 1000,
-			});
-
-			await expect(page.getByText('1', {exact: true}).nth(2)).toBeVisible(
+			await expect(page.getByTitle('Go to Dashboard Page')).toContainText(
+				pageName,
 				{
 					timeout: 100 * 1000,
 				}
 			);
-		});
-
-		await test.step('Delete pages created in DXP during automation execution', async () => {
-			await page.goto(liferayConfig.environment.baseUrl);
 
-			await apiHelpers.jsonWebServicesLayout.deleteLayout(
-				String(sitePage.id)
+			await expect(page.getByText('1', {exact: true}).nth(2)).toBeVisible(
+				{
+					timeout: 100 * 1000,
```

Comment:
@rafaellajordao do we really need a large timeout here? If not can you remove this? The default timeout is already 15s :smile: